copilot/fastapi/app/argo_cd_api.py

```
@@ -0,0 +1,94 @@
+    }
+    
+    # Forward all headers from the original request
+    for header_key, header_value in request.headers.items():
+        if header_key.lower() not in ["host", "connection", "content-length", "authorization"]:
+            headers[header_key] = header_value
+    
+    # Create target URL
+    target_url = f"{settings.argo_cd_url}/api/v1/{path}"
+    
+    # Pass through the request without modification
+    async with httpx.AsyncClient(verify=settings.verify_ssl) as client:
```

Ruff (0.8.2)

Redefinition of unused `client` from line 10

(F811)